controllers/infra/drivers/driver.go

```diff
@@ -30,10 +30,20 @@ import (
 type Driver interface {
 	CreateInstances(hosts *v1.Hosts, infra *v1.Infra) error
 	DeleteInstances(hosts *v1.Hosts) error
+	StopInstances(hosts *v1.Hosts) error
+	ModifyInstances(curHosts *v1.Hosts, desHosts *v1.Hosts) error
 	DeleteInstanceByID(instanceID string, infra *v1.Infra) error
 	GetInstancesByLabel(key string, value string, infra *v1.Infra) (*v1.Hosts, error)
 	// get infra all current hosts
 	GetInstances(infra *v1.Infra) ([]v1.Hosts, error)
+	// Volumes operation
+	// Create and Attach
+	CreateVolume(infra *v1.Infra, host *v1.Hosts, disks *v1.Disk) error
+	CreateVolumes(infra *v1.Infra, host *v1.Hosts, disks []v1.Disk) error
```

Comment:
Seems only need one of those functions. `CreateVolume` is `CreateVolume(xx,xx, 1 volume)`

Reply:
CreateVolumes create len(disks) goroutines to run CreateVolume. I think CreateVolume shouldn't be exposed.

Reply:
yes